study-builder/studies/rarex/study.conf

```diff
@@ -439,8 +440,25 @@
                 {
                   "type": "ACTIVITY_DATE",
                   "field": "date",
-                  "activityCode": "CONSENT"
-                }
+                  "activityCode": "CONSENT",
+                },
+                {
+                  "type": "TEXT",
+                  "field": "witness_printed_name",
+                  "activityCode": "CONSENT",
+                  "questionStableId": "CONSENT_IMPARTIAL_WITNESS_NAME"
+                },
+                {
+                  "type": "TEXT",
+                  "field": "witness_signature",
+                  "activityCode": "CONSENT",
+                  "questionStableId": "CONSENT_IMPARTIAL_WITNESS_SIGNATURE"
+                },
+                {
+                  "type": "ACTIVITY_DATE",
+                  "field": "date",
```

Comment:
I think we need to give the "Date of Signature" field for the witness a different name in the PDF.

Reply:
great catch! it should be `witness_date`